### DISCUSSION TITLE
How to load wasm as a componnet in js?

### DISCUSSION THREAD

**Discussion by SongOfYouth** [unoplatform/uno#18213] [Q&A] [answered]
Is it possible to invoke the wasm file in js such as vue?

**Comment by SongOfYouth** [comment]
Is it the solution at https://github.com/unoplatform/Uno.Wasm.Bootstrap/blob/main/doc/features-embedded.mode.md?

**Reply by jeromelaban** [reply, marked as answer]
At this time, it's not possible as .NET itself does not provide this capability.
